test/exlasticsearch/repo_test.exs

```diff
@@ -2,138 +2,81 @@ defmodule ExlasticSearch.RepoTest do
   use ExUnit.Case, async: true
 
   alias ExlasticSearch.Aggregation
-  alias ExlasticSearch.MultiVersionTestModel, as: MVTestModel
   alias ExlasticSearch.Query
   alias ExlasticSearch.Repo
-  alias ExlasticSearch.TestModel
-  alias ExlasticSearch.TestModel2
+  alias ExlasticSearch.TypelessMultiVersionTestModel, as: TypelessMVTestModel
+  alias ExlasticSearch.TypelessTestModel
 
-  setup_all do
-    Repo.delete_index(TestModel)
-    Repo.create_index(TestModel)
-    Repo.create_mapping(TestModel)
+  require Logger
 
-    Repo.delete_index(TestModel2)
-    Repo.create_index(TestModel2)
-    Repo.create_mapping(TestModel2)
+  setup_all do
+    Repo.delete_index(TypelessTestModel, :es8)
+    Repo.create_index(TypelessTestModel, :es8)
+    {:ok, %{status_code: 200}} = Repo.create_mapping(TypelessTestModel, :es8)
 
-    Repo.delete_index(MVTestModel)
-    Repo.create_index(MVTestModel)
-    Repo.create_mapping(MVTestModel)
+    Repo.delete_index(TypelessMVTestModel, :es8)
+    Repo.create_index(TypelessMVTestModel, :es8)
+    {:ok, %{status_code: 200}} = Repo.create_mapping(TypelessMVTestModel, :es8)
 
-    Repo.delete_index(MVTestModel, :read)
-    Repo.create_index(MVTestModel, :read)
-    Repo.create_mapping(MVTestModel, :read)
     :ok
   end
 
   describe "#index" do
-    test "It will index an element in es" do
-      model = %ExlasticSearch.TestModel{id: Ecto.UUID.generate()}
-      {:ok, _} = Repo.index(model)
+    @tag :elasticsearch8
+    test "It will index an element in es 8+" do
+      model = %ExlasticSearch.TypelessTestModel{id: Ecto.UUID.generate()}
+      {:ok, _} = Repo.index(model, :es8)
 
-      assert exists?(model)
+      assert exists?(model, :es8)
     end
   end
 
   describe "#update" do
-    test "It will update an element in es" do
+    @tag :elasticsearch8
+    test "It will update an element in es 8+" do
       id = Ecto.UUID.generate()
-      model = %ExlasticSearch.TestModel{id: id, name: "test"}
-      Repo.index(model)
+      model = %ExlasticSearch.TypelessTestModel{id: id, name: "test"}
+      Repo.index(model, :es8)
 
-      {:ok, _} = Repo.update(ExlasticSearch.TestModel, id, %{doc: %{name: "test edited"}})
+      {:ok, _} = Repo.update(ExlasticSearch.TypelessTestModel, id, %{doc: %{name: "test edited"}}, :es8)
 
-      {:ok, %{_source: data}} = Repo.get(model)
+      {:ok, %{_source: data}} = Repo.get(model, :es8)
       assert data.name == "test edited"
     end
   end
 
   describe "#bulk" do
-    test "It will bulk index/delete from es" do
-      model = %ExlasticSearch.TestModel{id: Ecto.UUID.generate()}
-      {:ok, _} = Repo.bulk([{:index, model}])
+    @tag :elasticsearch8
+    test "It will bulk index/delete from es 8+" do
+      model = %ExlasticSearch.TypelessTestModel{id: Ecto.UUID.generate()}
+      {:ok, _} = Repo.bulk([{:index, model, :es8}], :es8)
 
-      assert exists?(model)
-    end
-
-    test "It will bulk update from es" do
```

Comment:
Can we bring this test back for the typeless test model? This is one of the pieces of functionality massdriver actually depends on.